src/services/panthalassa/index.js

```diff
@@ -0,0 +1,318 @@
+ * @returns {Promise<*>} TODO
+ */
+export async function panthalassaConnectToDAppDevHost(address: String):Promise<boolean> {
+  return Panthalassa.PanthalassaConnectToDAppDevHost(address);
+}
+
+/**
+ * @desc TODO
+ * @param {string} rawMsg TODO
+ * @param {string} secretID TODO
+ * @param {string} secret TODO
+ * @param {string} receiverIdKey TODO
+ * @returns {Promise<*>} TODO
+ */
+export async function panthalassaCreateDAppMessage(rawMsg: String, secretID: String, secret: String, receiverIdKey:String):Promise<string> {
+  return Panthalassa.PanthalassaCreateDAppMessage({
+    rawMsg,
+    secretID,
+    secret,
+    receiverIdKey,
+  });
+}
+
+/**
+ * @desc TODO
+ * @param {string} identityPublicKey TODO
+ * @param {string} preKeyBundle TODO
+ * @returns {Promise<*>} {string} TODO
+ */
+export async function panthalassaInitializeChat(identityPublicKey: String, preKeyBundle: String):Promise<string> {
+  return Panthalassa.PanthalassaInitializeChat({
+    identityPublicKey,
+    preKeyBundle,
+  });
+}
+
+/**
+ * @desc TODO
+ * @param {string} message TODO
+ * @param {string} preKeyBundlePrivatePart TODO
+ * @returns {Promise<*>} {string} TODO
+ */
+export async function panthalassaHandleInitialMessage(message: String, preKeyBundlePrivatePart: String):Promise<string> {
+  return Panthalassa.PanthalassaHandleInitialMessage({
+    message,
+    preKeyBundlePrivatePart,
+  });
+}
+
+/**
+ * @desc TODO
+ * @param {string} rawMsg TODO
+ * @param {string} secretID TODO
+ * @param {string} secret TODO
+ * @param {string} receiverIdKey TODO
+ * @returns {Promise<*>} {string}
+ */
+export async function panthalassaCreateHumanMessage(rawMsg: String, secretID: String, secret: String, receiverIdKey: String):Promise<string> {
+  return Panthalassa.PanthalassaCreateHumanMessage({
+    rawMsg,
+    secretID,
+    secret,
+    receiverIdKey,
+  });
+}
+
+/**
+ * @desc TODO
+ * @param {string} message TODO
+ * @param {string} secret TODO
+ * @returns {Promise<*>} {string} TODO
+ */
+export async function panthalassaDecryptMessage(message: String, secret: String):Promise<string> {
+  return Panthalassa.PanthalassaDecryptMessage({
+    message,
+    secret,
+  });
+}
+
+/**
+ * @desc TODO
+ * @param {string} id TODO
+ * @param {string} msg TODO
+ * @param {string} context TODO
+ * @returns {Promise<*>} {string}
+ */
+export async function panthalassaRenderMessage(id: String, msg: String, context: String):Promise<string> {
+  return Panthalassa.PanthalassaRenderMessage({
+    id,
+    msg,
+    context,
+  });
+}
+
+/**
+ * @desc TODO
+ * @param {string} id TODO
+ * @param {string} data TODO
+ * @param {string} responseError TODO
+ * @param {string} timeout TODO
+ * @returns {Promise<*>} {string} TODO
+ */
+export async function panthalassaSendResponse(id: String, data: String, responseError: String, timeout: Number):Promise<boolean> {
+  return Panthalassa.PanthalassaSendResponse({
+    id,
+    data,
+    responseError,
+    timeout,
+  });
+}
```

Comment:
New methods are missing - PanthalassaSendMessage, PanthalassaSetLogger, etc. Please check that js interface is matching native one.

Reply:
Corrected and updated all methods